app/javascript/packs/application.js

```
@@ -3,7 +3,9 @@ import main from '../mastodon/main';
 if (!window.Intl || !Object.assign || !Number.isNaN ||
     !window.Symbol || !Array.prototype.includes) {
   // load polyfills dynamically
-  import('../mastodon/polyfills').then(main);
+  import('../mastodon/polyfills').then(main).catch(e => {
+    // console.log(e); // no-op
+  });
```

Doing a no-op just means it will swallow the error...